src/collectors/proc.plugin/sys_class_power_supply.c

```diff
@@ -116,14 +130,83 @@ static void add_labels_to_power_supply(struct power_supply *ps, RRDSET *st) {
     rrdlabels_add(st->rrdlabels, "device", ps->name, RRDLABEL_SRC_AUTO);
 }
 
+static bool read_simple_property(struct simple_property *prop, bool keep_fds_open) {
+    char buffer[30 + 1];
```

Comment:
Please, can you use a `define NAME` to express the meaning of number `30`? Thanks!